astroplan/scheduling.py

```diff
@@ -137,14 +137,13 @@ def create_score_array(self, time_resolution=1*u.minute):
         start = self.schedule.start_time
         end = self.schedule.end_time
         times = time_grid_from_range((start, end), time_resolution)
-        score_array = np.ones((len(self.blocks), len(times)))
+        score_array = u.Quantity(np.ones((len(self.blocks), len(times))))
         for i, block in enumerate(self.blocks):
             # TODO: change the default constraints from None to []
             if block.constraints:
                 for constraint in block.constraints:
-                    applied_score = constraint(self.observer, block.target,
+                    score_array[i] *= constraint(self.observer, block.target,
                                                times=times)
```

Comment:
We're getting a PEP8 compliance error because the indent of `times=times` is now misaligned with the `(` on the previous line. Could you add a commit that corrects the indentation of line 146?